k4Reco/DDPlanarDigi/components/DDPlanarDigi.cpp

```diff
@@ -101,7 +101,7 @@ std::tuple<edm4hep::TrackerHitPlaneCollection, edm4hep::TrackerHitSimTrackerHitL
   auto thsthcol  = edm4hep::TrackerHitSimTrackerHitLinkCollection();
 
   std::string cellIDEncodingString = m_geoSvc->constantAsString(m_encodingStringVariable.value());
-  dd4hep::DDSegmentation::BitFieldCoder bitFieldCoder(cellIDEncodingString);
+  dd4hep::DDSegmentation::BitFieldCoder bitFieldCoder("system:4,layer:4,ring:2,module:8,sensor:2,x:32:-12,y:-12");
```

Comment:
I suppose this should not be here? Or do we really want to hardcode an encoding string here?

Reply:
Argh this was a local change that I had for some reason but didn't want to merge, I'll make a commit to fix it